eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,8 +1,33 @@
+resources:
+  repositories:
+    - repository: azure-sdk-tools
```

Comment:
Do we even need azure-sdk-tools any longer? If so then we should add the ref here, otherwise lets just remove it.